bolt/include/bolt/Profile/YAMLProfileReader.h

```diff
@@ -56,6 +56,10 @@ class YAMLProfileReader : public ProfileReaderBase {
   /// is attributed.
   FunctionSet ProfiledFunctions;
 
+  /// Maps profiled function id to name, for function matching with calls as
+  /// anchors.
+  DenseMap<uint32_t, StringRef> IdToFunctionName;
```

Comment:
nit: use a type alias. Will make it easier to read specially when the type is used in an argument list.